libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -254,15 +255,15 @@ class MapboxTripSession(
 
     private fun updateRawLocation(rawLocation: Location) {
         locationObservers.forEach { it.onRawLocationChanged(rawLocation) }
+        lastLocationUpdateTimeMillis = Date().time
+        statusUpdateTimer.stopJobs()
         ioJobController.scope.launch {
             navigator.updateLocation(rawLocation)
-            lastLocationUpdateTimeMillis = Date().time
-            statusUpdateTimer.stopJobs()
-        }
-        fireOffStatusPolling()
-        statusUpdateTimer.startTimer {
-            if (Date().time - lastLocationUpdateTimeMillis >= STATUS_POLLING_INTERVAL) {
-                fireOffStatusPolling()
+            fireOffStatusPolling()
```

Comment:
Let's rename this method - it doesn't start the periodical polling anymore, it just gets the data once.